src/test/java/utilities/Util.java

```diff
@@ -15,4 +18,14 @@ public static void wait(int seconds) {
 		}
 	}
 
+   /*
+    * Selectdate
+    * 
+    * @Author:Yiping Chen
+    */
+	public static void selectdate(WebDriver driver,String calendarLocator) {
```

Comment:
1, Instead of passing in a xpath as parameter, it would be more reusable to use a `By` object
2, Formatting is off